src/fshtypes/Configuration.ts

```diff
@@ -36,7 +36,8 @@ export type Configuration = {
   contained?: any[];
   extension?: Extension[];
   modifierExtension?: Extension[];
-  url: string;
+  canonical: string;
+  url?: string;
```

Comment:
Sorry if this is an annoying request, but could you put canonical directly below `filePath`? Or nearer the end of the definition of `Configuration`. I ask because right now these properties match up against the properties of the IG resource perfectly in the same order.